From f6b2c4961570130e994eaab5d53ee091ded4d712 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 2 Oct 2025 05:19:55 -0700
Subject: [PATCH] update argminmax version and features

---
 downsample_rs/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/downsample_rs/Cargo.toml b/downsample_rs/Cargo.toml
index 994e8d5..1a752f4 100644
--- a/downsample_rs/Cargo.toml
+++ b/downsample_rs/Cargo.toml
@@ -9,7 +9,7 @@ readme = "README.md"
 
 [dependencies]
 # TODO: perhaps use polars?
-argminmax = { version = "0.6.1", features = ["half"] }
+argminmax = { version = "0.6.2", default-features = false , features = ["float", "half"] }
 half = { version = "2.3.1", default-features = false , features=["num-traits"], optional = true}
 num-traits = { version = "0.2.17", default-features = false }
 once_cell = "1"
-- 
2.39.3 (Apple Git-146)

