From 1c2d067697cdd737bf18d5af076e9db345a05006 Mon Sep 17 00:00:00 2001
From: Andrii Vetroumov <avetroumov@anaconda.com>
Date: Wed, 8 Jul 2026 09:32:03 +0300
Subject: [PATCH] update argminmax

---
 downsample_rs/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/downsample_rs/Cargo.toml b/downsample_rs/Cargo.toml
index 81aafd2..ed66f0d 100644
--- a/downsample_rs/Cargo.toml
+++ b/downsample_rs/Cargo.toml
@@ -8,7 +8,7 @@ license = "MIT"
 
 [dependencies]
 # TODO: perhaps use polars?
-argminmax = { version = "0.6.3", features = ["half"] }
+argminmax = { version = "0.6.2", default-features = false , features = ["float", "half"] }
 half = { version = "2.7.1", default-features = false , features=["num-traits"], optional = true}
 num-traits = { version = "0.2.19", default-features = false }
 once_cell = "1.21"
-- 
2.54.0

